### PR TITLE
Fixing bug replacing {char} with player name

### PR DESCRIPTION
**Pull request by moabdelbasset**

### CODE REVIEW

File: assets/js/script.js [hunk 1]
```diff
@@ -129,10 +129,10 @@ const handleChoice = async (choiceIndex) => {
   let result = selectedStory[currentSceneKey].choices[choiceIndex].result;
   document.getElementById("storyboard").innerText = `${action}. ${result}`;
 
-  await new Promise(resolve => setTimeout(resolve, 6000));
+  //await new Promise(resolve => setTimeout(resolve, 6000));
```
Comment (Dee-McG) [inline assets/js/script.js:132]: Can you uncomment these so we dont add broken code to main please

File: assets/js/script.js [hunk 2]
```diff
@@ -309,6 +309,7 @@ document.getElementById("set-character-name-btn").addEventListener("click", func
  */
 function replaceCharWithCharacterName(text) {
   const charName = sessionStorage.getItem("characterName") || "DefaultName";
+  console.log("Replacing {char} with:", charName);
```
Comment (Dee-McG) [inline assets/js/script.js:312]: Remove

File: index.html [hunk 3]
```diff
@@ -14,8 +14,22 @@
       href="https://fonts.googleapis.com/css2?family=Creepster&display=swap"
       rel="stylesheet"
     />
+    <link rel="icon" type="image/x-icon" href="assets/icons/icons8-halloween-64.png">
   </head>
   <body>
+    <header>
+      <div class="logo">
+        <a href="/index.html">
```
Comment (Dee-McG) [inline index.html:22]: This wont load on deployed site, remove the /

File: index.html [hunk 4]
```diff
@@ -110,12 +124,18 @@ <h1 id="story-selection-heading">Select Your Story!</h1>
           <p class="hidden">Thank you for taking the time to try out our creation, we hope you have enjoyed the experience and pass the link onto your friends and family. We hope you will come back and try out another of our stories. Have a spooktacular day.</p>
         </div>
         <div class="btn-container">
-          <a class="home-btn" href="/index.html">Return home</a>
-          
-            
+          <a class="home-btn hidden" href="/index.html">Return home</a>
```
Comment (Dee-McG) [inline index.html:127]: Remove /

File: assets/js/script.js [hunk 5]
```diff
@@ -129,10 +129,10 @@ const handleChoice = async (choiceIndex) => {
   let result = selectedStory[currentSceneKey].choices[choiceIndex].result;
   document.getElementById("storyboard").innerText = `${action}. ${result}`;
 
-  await new Promise(resolve => setTimeout(resolve, 6000));
+  //await new Promise(resolve => setTimeout(resolve, 6000));
   loadEpilogue(selectedStory[currentSceneKey]);
 
-  await new Promise(resolve => setTimeout(resolve, 10000));
+  //await new Promise(resolve => setTimeout(resolve, 10000));
```
Comment (Dee-McG) [inline assets/js/script.js:135]: Can you uncomment these so we dont add broken code to main please

Comment (moabdelbasset) [reply assets/js/script.js:135]: done

File: assets/js/script.js (hunk 1 above)
Comment (moabdelbasset) [reply assets/js/script.js:132]: done

**Review by Dee-McG — APPROVED** [review]

**Pull request merged**